pom.xml

```diff
@@ -20,8 +20,8 @@
 		<java.version>17</java.version>
 		<maven.compiler.target>17</maven.compiler.target>
 		<maven.compiler.source>17</maven.compiler.source>
-		<spring-cloud.version>2022.0.2</spring-cloud.version>
-		<snakeyaml.version>2.0</snakeyaml.version>
+		<spring-cloud.version>2022.0.3</spring-cloud.version>
```

Comment:
Recently was released also the 2022.0.4 if I'm not wrong 🧐

Reply:
We'll consider the update in the future, ATM the new version triggers some errors in the tests.